src/components/Layout.js

```diff
@@ -1,12 +1,11 @@
 import { Outlet, NavLink } from 'react-router-dom';
-import { BsPersonFill } from 'react-icons/bs';
 
 const Layout = () => {
   const navLinkStyle = ({ isActive }) => {
     const activeLinkStyle = {
       fontWeight: isActive ? 'bold' : 'normal',
       textDecoration: isActive ? 'underline' : 'none',
-      color: isActive ? '#7c0808' : '#a8a8a8',
+      color: isActive ? '#7c1f1f' : '#0e0ae6',
     };
     return activeLinkStyle;
```

Comment:
- [ ] Please let's make the navbar identical to the given zeplin design as seen below 👇 
## Zeplin Design Guidelines
![image](https://user-images.githubusercontent.com/46839548/218284671-85a5dabf-2269-42af-873e-06f33d1a4f8b.png)
`
## Your Design
![image](https://user-images.githubusercontent.com/46839548/218284773-4f39d7d0-0ab1-48b9-9b73-23f41d4f7064.png)